misk/src/main/kotlin/misk/web/HttpCall.kt

```diff
@@ -31,7 +31,12 @@ interface HttpCall {
   val requestHeaders: Headers
 
   /** The HTTP response under construction. */
+  /** Meaningful HTTP status about what actually happened */
```

Comment:
nit: remove old javadoc comment, and add comment about how this may not be the code actually sent over the network and why this is the case. I see it's commented elsewhere, but adding the comment here too will help reduce confusion